packages/ui/package.json

```diff
@@ -1,24 +1,87 @@
 {
   "name": "@penumbra-zone/ui",
   "version": "3.5.0",
-  "private": true,
+  "private": false,
   "license": "(MIT OR Apache-2.0)",
   "type": "module",
+  "engine": {
+    "node": ">=22"
+  },
   "scripts": {
+    "build": "tsup",
     "build-storybook": "storybook build",
+    "clean": "rm -rf dist package penumbra-zone-*.tgz",
     "lint": "eslint components lib",
+    "prepack": "$npm_execpath run build",
```

Comment:
question: do we need `prepack`? The `packages-release.yml` workflow defines a build script with Turbo, so the build of a package would probably happen twice increasing the CI pipeline time.

Reply:
it's nicer when using from cli

Reply:
tsconfig now uses `incremental`, and without api-extractor running the slowdown is minimal

Reply:
removed the `prebuild` script as suggested